src/lipo.rs

```diff
@@ -5,6 +5,18 @@ use failure::ResultExt;
 use log::info;
 use std::fs;
 use std::process::Command;
+use std::path::Path;
+
+fn is_output_updated(output: impl AsRef<Path>, inputs: impl IntoIterator<Item=impl AsRef<Path>>) -> std::io::Result<bool> {
```

Comment:
I think you mean `is_output_up_to_date`? (The current function name implies the opposite to me).

Alternatively, `should_update_output` might be even clearer.